components/caseStudyDetails/ChallengeLearning.tsx

```diff
@@ -0,0 +1,61 @@
+import { Card } from "@nextui-org/card";
+import Image from "next/image";
+
+const ChallengeLearning = ({
+  challengesOrLearnings,
+  bulletPoints,
+  icon,
+}: {
+  challengesOrLearnings: string;
+  bulletPoints: string[];
+  icon: string;
+}) => {
+  return (
+    <Card className="challengesLearnings border-0 shadow-none dark:bg-black300 sm:gap-[1.88rem] sm:px-10 sm:py-9">
+      <h6
+        className={`bodyBold ${
+          challengesOrLearnings === "CHALLENGES"
+            ? "text-[#E15A46]"
```

Comment:
Please don't hardcode the colors in your code, put them in your tailwind config.

Reply:
Hi @Nassim-Saboundji, I put them in my tailwind config.